controllers/virtualmachinepublishrequest/virtualmachinepublishrequest_controller.go

```diff
@@ -673,10 +840,17 @@ func (r *Reconciler) ReconcileNormal(ctx *context.VirtualMachinePublishRequestCo
 			ctx.Logger.Error(err, "failed to publish VirtualMachine")
 			return ctrl.Result{}, errors.Wrapf(err, "failed to publish VirtualMachine")
 		}
-		return ctrl.Result{RequeueAfter: requeueDelay(ctx)}, nil
```

Comment:
Is it clearer to keep this return in here or to move the following checks in an `} else {` block since AFAICT none of the following checks could be true on an async pub req